index.bs

```diff
@@ -4959,7 +4960,13 @@ partial interface MLGraphBuilder {
 <dl dfn-type=dict-member dfn-for=MLReduceOptions>
     : <dfn>axes</dfn>
     ::
-        The dimensions to reduce. The values in the list must be in the range [0, N-1] where N is the [=MLOperand/rank=] of the input tensor. If not present, all dimensions are reduced. If empty, no dimensions are reduced, and the shape of the output tensor is the same as the shape of the input tensor.
+        The dimensions to reduce, which also specifies which of the values in the input tensor are used as input values to the reduction function. The values in the list must be in the range [0, N-1] where N is the [=MLOperand/rank=] of the input tensor.
```

Comment:
```suggestion
        The dimensions to reduce, which also specifies which of the values in the input tensor are used with the reduction function. The axes in the list must be in the range [0, N-1] where N is the [=MLOperand/rank=] of the input tensor.
```
Propose avoiding the word "values" for the axis list because "values" was just used in the previous sentence in a very different context, meaning the elements inside the input tensor, whereas values here actually means the axes. Alternately, "The axis values must be..." could work.

Little repetitive: "...which of the *values* in the *input* are used as *input* *values*...".

Reply:
My intention here was to define (casually, not a full `<dfn>`) the "input values" term, which would then be used later on. And make it explicit that there are values that go through the reduction function, even if there are no axes specified. But yes,"values" here is confusing.

Obviously I wasn't successful. :stuck_out_tongue: